device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/Message.java

```diff
@@ -63,6 +64,7 @@
     /**
      * [Optional] On C2D messages it is interpreted by hub to specify the expected feedback messages. Ignored in other cases.
      */
+    @SuppressWarnings("FieldCanBeLocal")
     private FeedbackStatusCodeEnum ack;
```

Comment:
This is assigned in initializer method for a new message.

Reply:
This field isn't used anywhere, so you should just remove it

Reply:
Do you think same for feedbackStatusCode?
I don't see any references for that one as well.